wordpress-mobile/WordPress-iOS
# Add a baseline set of internal icons

Fixes missing image warnings when checking out `develop` for the first time.

**To test:**
- Check out this branch
- Ensure there are no missing image warnings.

**Update release notes:**
